dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/WorkerServer.java

```diff
@@ -128,26 +107,4 @@ public void stop(String cause) {
         close(cause);
     }
 
-    public void killAllRunningTasks() {
```

Comment:
I think we keep this method to make sure all task are canceled when worker down, WDYT?

Reply:
Ok, I keep this method, and add todo.